src/layouts/AppShellLayout/AppShellLayout.tsx

```diff
@@ -0,0 +1,16 @@
+import TopBar from "../TopBar";
+
+type AppShellLayoutProps = {
+  children: React.ReactNode;
+};
+
+const AppShellLayout = ({ children }: AppShellLayoutProps) => {
```

Comment:
Starting from React 18, the automatic inclusion of children as a prop in React.FC has been deprecated. Since I'm using React version ^18.3.1 in my project, this behavior no longer works as expected. Additionally, React.FC has been removed from the Create React App TypeScript template, making it no longer available by default.

https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components